app/pub_sub/subscribers/site_activity.rb

```diff
@@ -0,0 +1,33 @@
+module Subscribers
+  class SiteActivity < ::Subscribers::Base
+    def site_created(event)
+      return if event.payload[:subject].nil? || event.payload[:subject].new_record?
+
+      create_activity_from_event(event)
+    end
+
+    def site_updated(event)
+      return if event.payload[:changes].empty? || !event.payload[:subject].valid?
+
+      create_activity_from_event(event)
+    end
+
+    def site_deleted(event)
+      return if !event.payload[:subject].destroyed?
+
+      create_activity_from_event(event)
+    end
+
+    private
+
+    def create_activity_from_event(event)
```

Comment:
Yes, that´s something I suspect we´lll be adding in the closer future, once we start to render the activity and see how it looks. For the moment I was just skipping this until we face the wall.